source/FreeRTOS_IP_Utils.c

```diff
@@ -321,6 +321,10 @@ BaseType_t xIsCallingFromIPTask( void )
 /**
  * @brief Process a 'Network down' event and complete required processing.
  */
+/* MISRA Ref 8.9.1 [File scoped variables] */
+/* More details at: https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/main/MISRA.md#rule-89 */
+/* coverity[misra_c_2012_rule_8_9_violation] */
+/* coverity[single_use] */
```

Comment:
Does this suppress the violation in the report? If I remember correctly, we had found that doing this doesn't help.

Reply:
It does suppress the violation in the report